README.md

```diff
@@ -100,6 +106,41 @@ General security best practices apply:
+
+## Step 1: Identify valuable assets
+
+This is probably a lot of things: your phone, your laptop, passwords stored on your devices, internet browsing history, etc. Make a list starting with the most important assets to protect. You can put them in categories based on how important they are: public, sensitive, or secret.
+
+## Step 2: Identify adversaries
+
+Define whom you are defending against. Start by defining the motivation they might have to attack your assets. [Financial gain](https://www.verizon.com/business/resources/reports/dbir/) is a big motivator for many attackers, for example.
+
+## Step 3: Identify the adversaries' capabilities
+
+In order to counter your adversaries, you'll need to understand what they're capable of and what they're not capable of. Rank adversaries from totally usophisticated to very advanced. For example, a common thief is not very sophisticated; they will likely be stopped by basic things like simply having a password and drive encryption on your device. A very advanced adversary like a state actor might require fully turning off your device when not in use to clear the keys from RAM and a long diceware password.
+
+## Step 4: Come up with ways to mitigate the threats
+
+Now is when you decide the best way to counter each threat. You might avoid writing passwords down on paper so your roommate can't find them or you might encrypt the drive on your computer so a thief can't get data from it.
+
+## Step 5: Implement your plan
+
+Start implementing your strategies. Depending on how serious or time-sensitive your situation is, you might want to ease into it so you don't burn yourself out too quickly. Sometimes you might find that a strategy you came up with isn't working for whatever reason, that's ok. Just go back and rethink that part. Eventually you will get to a workable plan that mitigates your threats and that doesn't cause undue inconvenience on your life.
+
+Here's an example of the type of table you should make for each asset you want to protect:
+
+| Adversary | Motivation | Capabilities | Mitigation |
+| --- | --- | --- | --- |
+| Nation State | See contents of packets sent over the internet | Passive surveillance of internet infrastructure, advanced computers for cracking encryption/analysis of packets | Always use encryption for all connections |
+| Hacker | Infect my device with malware to steal personal info and/or money | Run unauthorized code on my computer to steal my info | Use sandboxing, enable security features in my OS, keep OS and all software updated and turn on autoupdates |
```

Comment:
Hacker is too generic. Call this adversary the Criminal, and their motivation is primarily Financial. Capabilities include social engineering, cots malware, password reuse, etc. (check the Verizon report for some details to include).

Reply:
done